api-client/src/utils/index.ts

```diff
@@ -1,7 +1 @@
-export const getSlugFromPath = (path: string): string => {
-  const slug = path.split('/').pop()
-  if (!slug) {
-    throw new Error('Empty string is not a valid path')
-  }
-  return slug
-}
+export const getSlugFromPath = (path: string): string => path.split('/').pop()!
```

Comment:
why do we always have a slug?

Reply:
Split always returns an array of length >= 1, even for an empty string this results in an empty string, therefore it will never be undefined.